packages/carbon-graphs/CHANGELOG.md

```diff
@@ -7,3 +7,6 @@
 * Changed
   * Added a condition to check if graphContainer is present and then resize the graph.
   * Migrated to the terra-graphs mono-rep.
+
+* Changed
+  * For consistency replaced `document` in all draw helpers with `graph.legendSVG` for better control.
```

Comment:
This should be under the `Unreleased` heading

Reply:
But didn't we want it below `Migrated to the terra-graphs mono-rep.` in-order to maintain chronological order?

Reply:
Oh, when I made that suggestion, that line was also in the `Unreleased` section. However we did a new release afterwards, so `Migrated to the terra-graphs mono-rep` got moved to the `2.15` section. But any changes we make will always go to the `Unreleased` section because we are no longer releasing after every commit. Sorry for the confusion.

Reply:
updated here https://github.com/cerner/terra-graphs/pull/18/commits/30019d1a1afbecf20ff1b8476b29e225df1cc68b